auth/src/VSCodeAzureSubscriptionProvider.ts

```diff
@@ -58,8 +58,9 @@ export class VSCodeAzureSubscriptionProvider extends vscode.Disposable implement
      *
      * @returns A list of tenants.
      */
-    public async getTenants(): Promise<TenantIdDescription[]> {
-        const { client } = await this.getSubscriptionClient();
+    public async getTenants(account?: vscode.AuthenticationSessionAccountInformation): Promise<TenantIdDescription[]> {

```

Comment:
What's the behavior when we don't pass the account arg here? I know that was the previous behavior, but I still have no idea what's supposed to happen. I mostly want to know so we can document it.

Reply:
Follow up: Althought it'd be a breaking change, it might be worth investigating making the account arg required. Especially if the behavior of not passing it in is unknown/indeterminate.

Reply:
I think that was a passed behavior. I'll work on making it a required property.

Reply:
With your recent changes I think it's now possible to make this required

Reply:
But account is not always passed in? That's the whole reason we are checking each account to get all the tenants?

Reply:
Oh duh